### PR TITLE
fix(blade): typo in exports field

### LINKED ISSUE #387: [Blade/New] Typo in exports in package.json
## What is this issue about?

There is a [typo](https://github.com/razorpay/blade/blob/master/packages/blade/package.json#L25) in exports field in package.json for the `./tokens` package.

This causes issues while bundling code for webpack 5 as webpack 5 supports export maps defined in package.json. 
This is not an issue when working with webpack 4.

## Proposed Fix
Fix the typo by changing `deafult -> default`

### PR DESCRIPTION
**Pull request by sidag95**
Fixes typo in exports field in package.json

Fixes #387

### CODE REVIEW

File: packages/blade/package.json [hunk 1]
```diff
@@ -88,4 +88,4 @@
   "dependencies": {
     "@babel/runtime": "7.14.6"
   }
-}
+}
```
Comment (chaitanyadeorukhkar) [inline packages/blade/package.json:91]: New line :P

Comment (sidag95) [reply packages/blade/package.json]: Done

**Comment by sidag95** [comment]
@chaitanyadeorukhkar Have made the requested changes. What is the process of getting this merged?

**Review by dhruvdutt — APPROVED** [review]

**Comment by sidag95** [comment]
@dhruvdutt @chaitanyadeorukhkar Can I add a changeset to this PR or will one of you do that?

**Comment by sidag95** [comment]
@dhruvdutt @chaitanyadeorukhkar Added changeset, can you folks please approve this PR again?

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Pull request merged**